llvm/test/lab2/alexseev_danila/test.ll

```diff
@@ -0,0 +1,91 @@
+; RUN: opt -load-pass-plugin=%llvmshlibdir/AlexseevLoopPlugin%shlibext -passes=alexseev-loop-plugin -S %s | FileCheck %s
+

```

Comment:
Please add a test for a case when function with loops already has loop bordering calls.